pandas/tests/indexes/timedeltas/test_arithmetic.py

```diff
@@ -297,7 +297,7 @@ def test_dti_mul_dti_raises(self):
 
     def test_dti_mul_too_short_raises(self):
         idx = self._holder(np.arange(5, dtype='int64'))
-        with pytest.raises(ValueError):
```

Comment:
why did this change?

Reply:
Because the problem is now caught in TimedeltaIndex when checking what it is operating against, whereas before the problem was caught in numpy when trying to operate on differently-sized arrays.